docs/admin_api/rooms.md

```diff
@@ -511,3 +512,55 @@ optionally be specified, e.g.:
         "user_id": "@foo:example.com"
     }
 ```
+
+# Forward Extremities Admin API
+
+Enables querying and deleting forward extremities from rooms. When a lot of forward
+extremities accumulate in a room, performance can become degraded.
```

Comment:
Is there a reference issue for this or anything that we should link to?

Reply:
Will add a link to https://github.com/matrix-org/synapse/issues/1760